tox.ini

```diff
@@ -94,7 +94,7 @@ envlist =
     {py3.6,py3.9}-django-v{2.2}
     # - Django 3.x
     {py3.6,py3.9}-django-v{3.0}
-    {py3.6,py3.11}-django-v{3.2}
+    {py3.6,py3.9,py3.11}-django-v{3.2}
```

Comment:
Why add `3.9` here?

Reply:
to improve code coverages coverage reports. we need a python version where tests for Django 3.0 and Django 3.2 are running, else codecov always tells us that one part of a `if django_version < 3.2` in 
`sentry_sdk/integrations/django/caching.py` is not covered by tests.